cast/src/launcher/layout/hc-cast.ts

```diff
@@ -83,34 +83,33 @@ class HcCast extends LitElement {
               `
             : html`
                 <div class="section-header">PICK A VIEW</div>
-                <paper-listbox
-                  attr-for-selected="data-path"
-                  .selected=${this.castManager.status.lovelacePath || ""}
-                >
+                <mwc-list>
                   ${(
                     this.lovelaceViews ?? [
                       generateDefaultViewConfig({}, {}, {}, {}, () => ""),
                     ]
                   ).map(
-                    (view, idx) => html`
-                      <paper-icon-item
+                    (view, idx) =>
+                      html`<ha-list-item
+                        graphic="avatar"
                         @click=${this._handlePickView}
```

Comment:
We should not use a click handler here, as it should also work when navigating by keyboard.